.github/workflows/test.yml

```diff
@@ -0,0 +1,105 @@
+name: Test SkyPortal
+
+on: [push]
+
+jobs:
+  test:
+    name: Test SkyPortal
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11.8
+        env:
+          POSTGRES_PASSWORD: anything
+        ports:
+        - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - name: Lint
+        run: |
+          pip install pre-commit
+          if pre-commit run --from-ref origin/master --to-ref HEAD; then
+              echo "Linting errored; this will be fatal sometime in the near future"
+          fi
+
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.8'
```

Comment:
We should also be checking 3.7 (I believe we require >=3.7).

Reply:
Switched to 3.7 to test our minimal version of Python.